myVK/myVK/ViewController/Controller/Collection/FriendPhotoCollectionViewController.swift

```diff
@@ -9,6 +9,10 @@ final class FriendPhotoCollectionViewController: UICollectionViewController {
 
     var user: User?
 
+    // MARK: - Private Methods
+
+    private var networkService = VKAPIService()
```

Comment:
```suggestion
    private let networkService = VKAPIService()
```
либо networkService = NetworkService() (этот нагляднее)
либо vkApiService = VKAPIService()